packages/rancher-istio/charts/templates/istio-cni-psp.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: psp-istio-cni
```

Comment:
Wondering if we should expose this in `values.yaml`
(and its references throughout the other file(s))

Reply:
Do you mean exposing a field that allows you to enable or disable deploying PSPs? I kinda agree with this idea since I think this is what rancher-monitoring does too:

https://github.com/rancher/charts/blob/aae93d0125fad561024b25ac1a15580ea612e158/charts/rancher-monitoring/rancher-monitoring/9.4.203/values.yaml#L330

Reply:
I mean both the naming and that :)
I think it may be a good idea.